File: Solutions/6375/c++/program.cpp
// Copy from Quera

#include <iostream>

using namespace std ;

int main()
{
	long long a , b , c ;
	cin >> a >> b >> c ;
	if (a == b && b == c)
	{
		cout << 0 << endl ;
	}
	else if (abs(a - b) == abs(b - c) || abs(a - c) == abs(b - c) || abs(a - b) == abs(a - c) || (a == b && a != c && b != c) || (a == c && a != b && b != c) || (b == c && a != b && a != c))
	{
		cout << 1 << endl ;
	}
	else
	{
		cout << 2 << endl ;
	}
	return 0 ;
}

File: Solutions/6375/java/program.java
// Copy from Quera

import java.util.*;
import java.lang.Math;

class Main
{
	public static void main (String[] args)
	{
	    Scanner input = new Scanner(System.in);
	    long a = input.nextLong(), b = input.nextLong(), c = input.nextLong();
		if (a == b && b == c)
    		System.out.println("0");
    	else if (Math.abs(a - b) == Math.abs(b - c) ||
            	Math.abs(a - c) == Math.abs(b - c) ||
            	Math.abs(a - b) == Math.abs(a - c) ||
            	(a == b && a != c && b != c) ||
            	(a == c && a != b && b != c) ||
            	(b == c && a != b && a != c))
    	    System.out.println("1");
    	else
    	    System.out.println("2");
	}
}
